wikitemplate.md

```diff
@@ -168,9 +112,39 @@
 - [ ] Test zoom saved when you navigate within a single origin site
 - [ ] Test that navigating to a different origin resets the zoom
 
+## Printing
+
+- [ ] Test that you can print a PDF
+
+## Extensions/Plugins tests
+
+- [ ] Enable each extension one by one under `about:preferences#extensions` and ensure that the browser doesn't become unresponsive
+### Widevine
+  - [ ] Test that you can log into Netflix and start a show
+### Flash tests
+  - [ ] Test that flash placeholder appears on http://www.homestarrunner.com
+  - [ ] Test with flash enabled in preferences, auto play option is shown when visiting http://www.homestarrunner.com
+
+## Autofill tests
+
+- [ ] Test that autofill works on http://www.roboform.com/filling-test-all-fields
+- [ ] Verify clicking submit button on a form doesn't crash the browser
```

Comment:
I'm not seeing a `Submit` button anywhere on the page. Just a `Reset` button which isn't crashing `0.22.667`. We might want to use one of the examples from known cases that the community has provided us with.